### ISSUE TITLE
`--accept-dns` in deployment may be misleading 

### ISSUE THREAD

**Issue by arcoast** [GoodiesHQ/headscale-admin#23]
Currently `--accept-dns` adds the flg to the deployment command, however checking this on the command line with

```bash
❯ tailscale up --accept-dns --help
USAGE
  up [flags]

"tailscale up" connects this machine to your Tailscale network,
triggering authentication if necessary.

With no flags, "tailscale up" brings the network online without
changing any settings. (That is, it's the opposite of "tailscale
down").

If flags are specified, the flags must be the complete set of desired
settings. An error is returned if any setting would be changed as a
result of an unspecified flag's default value, unless the --reset flag
is also used. (The flags --auth-key, --force-reauth, and --qr are not
considered settings that need to be re-specified when modifying
settings.)

FLAGS
  --accept-dns, --accept-dns=false
        accept DNS configuration from the admin panel (default true)
```
reveals the default is `true` so I'm wondering if it might be better to either make a switch to confgure this to either `true` or `false` or perhaps alternatively relabel it to not accept DNS and then make the flag in the deployment command `--accept-dns=false`

**Comment by GoodiesHQ** [comment]
Thank you for this! This is definitely a good idea since `--accept-dns` is default behavior. I will change the label to "Ignore DNS Settings" or something similar and that will set `--accept-dns=false`. Much appreciated!

0.1.12 will be out in the coming days/weeks (not months, I promise lol).

**Comment by GoodiesHQ** [comment]
This has been addressed and will be released shortly. I ended up keeping the existing terminology. Because I am not certain that the default value will always stay that way, I left the "Accept DNS" default value checked which appends `--accept-dns`, and when unchecked, it appends `--accept-dns=false`. Thank you again! I'll reply again and close when 0.1.12 is released.

**Comment by arcoast** [comment]
@GoodiesHQ This is your baby, and that thinking makes perfect sense, appreciate all the hard work and the consideration.

**Comment by GoodiesHQ** [comment]
This has been pushed as the current `goodieshq/headscale-admin:dev` image and will be versioned in the coming days!

**Comment by arcoast** [comment]
> This has been addressed and will be released shortly. I ended up keeping the existing terminology. Because I am not certain that the default value will always stay that way, I left the "Accept DNS" default value checked which appends `--accept-dns`, and when unchecked, it appends `--accept-dns=false`. Thank you again! I'll reply again and close when 0.1.12 is released.

I've been thinking about this a bit more and I think, if my understanding is correct, that you are 110% correct with this approach, reason being in the headscale `config.yml` this may affect the default behaviour.  Not too sure, but seems logical that it could affect things

```yaml
dns_config:
  # Whether to prefer using Headscale provided DNS or use local.
  override_local_dns: true
  ```

**Comment by GoodiesHQ** [comment]
Good catch! I figured the default behavior may change but I'd forgotten about that setting entirely.

As this is now fixed in 0.1.12b-dev (and soon 0.1.12b) I'm going to close it, but please feel free to open anything else!
